packages/inquirer-common/README.md

```diff
@@ -0,0 +1,11 @@
+# @sap-ux/inquirer-common
+
+Commonly functionality used by inquirer modules for example `@sap-ux/ui5-application-inquirer`, `@sap-ux/ui5-library-inquirer` 
+
```

Comment:
would be nice to add a little more info about what this module includes

Reply:
Done thx

Reply:
Resolving with agreement